src/pages/sessions/running.js

```diff
@@ -29,13 +34,74 @@ class Running extends Component {
 
     return (
       <TeacherLayout intl={intl} navbar={navbarConfig} sidebar={{ activeItem: 'runningSession' }}>
-        <Grid padded columns="2">
-          <Grid.Column>blebleble</Grid.Column>
-          <Grid.Column>blablabla</Grid.Column>
-        </Grid>
+        <div className="runningSession">
+          <div className="sessionProgress"><SessionProgress intl={intl} /></div>
+          <div className="feedback">
+            <div className="confusionBarometer">
+              <ConfusionBarometer data={data.allUsers[0].activeSession.confusion} intl={intl} />
+            </div>
+            <div className="feedbackChannel">
+              <FeedbackChannel data={data.allUsers[0].activeSession.feedbacks} intl={intl} />
+            </div>
+          </div>
+        </div>
+
+        <style jsx>{`
+        .runningSession {
+          display: flex;
+          flex-direction: row;
+          flex-wrap: wrap;
+          margin: 1rem 6rem
+        }
+        .sessionProgress {
+          flex: 1 1 100%;
+          padding: 0.5rem
+        }
+        .feedback {
+          display: flex;
+          flex-flow: row wrap;
+          width: 100%;
```

Comment:
As `.feedback` is a child of `.runningSession`, which is `display: flex`, we might use `flex: 0 0 100%;` instead of `width: 100%`. It would not even be necessary to have this wrapper, as flexbox will align its children as they fit.